src/OrchardCore.Modules/OrchardCore.Media/Recipes/MediaStep.cs

```diff
@@ -32,6 +43,13 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
 
             foreach (var file in model.Files)
             {
+                if (!_allowedFileExtensions.Contains(Path.GetExtension(file.TargetPath), StringComparer.OrdinalIgnoreCase))
+                {
+                    _logger.LogWarning("File extension not allowed: '{0}'", file.TargetPath);
```

Comment:
```suggestion
                    _logger.LogWarning("File extension not allowed: '{Path}'", file.TargetPath);
```

Reply:
Done (and fixed the other place I cut and pasted from)